datajunction-server/datajunction_server/sql/parsing/ast.py

```diff
@@ -2296,9 +2296,15 @@ def __str__(self) -> str:
             if self.column_list
             else ""
         )
-        column_list_str = (
-            f"{cols}" if self.name.name.upper() != "UNNEST" else f"({cols})"
-        )
+
+        column_parens = False
+        if self.name.name.upper() == "UNNEST" or (
```

Comment:
Hmm, does this still work for `LATERAL VIEW EXPLODE`, which is only valid without parentheses, i.e. `LATERAL VIEW EXPLODE(somemap) AS col1, col2`?

Reply:
Yes it does still work in that case. If it's an `EXPLODE` function call I check whether the immediate parent in the ast is a `LateralView`:
```py
self.name.name.upper() == "EXPLODE" and not isinstance(self.parent, LateralView)
```
so it would include parens when it's just an `EXPLODE` of a map in a projection but would not include parens if it's an `EXPLODE` as part of a `LATERAL VIEW EXPLODE` in a from clause.

Reply:
Sounds good, thanks!